src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -872,7 +884,29 @@ public void deleteExperiment(String requestID, long experimentId, long projectId
 		}
 
 	}
+	/**
+	 * @param requestID
+	 * @param projectId
+	 */
+	public void isPersisted(String requestID, long projectId)
+	{
+
+		IGeppettoProject geppettoProject = retrieveGeppettoProject(projectId);
 
+		if(geppettoProject != null)
+		{
+			boolean persisted = !geppettoProject.isVolatile();
```

Comment:
Second time I'm seeing the persisted boolean being created.  Maybe it could be a method that gets called on geppettoProject that maps to the same thing (i.e. geppettoProject.isPersisted() {return !this.isVolatile() } )